### PR TITLE
Fix onChange trigger

### PR DESCRIPTION
**Pull request by s-horiguchi**

### CODE REVIEW

File: modules/Endpoints.js [hunk 1]
```diff
@@ -118,10 +115,14 @@ class Endpoints extends Component {
     })).reduce((prev, curr) => ({ ...prev, ...curr }))
 
     Object.keys(data).forEach(name => {
-      configs[name].onChange(data[name])
+      if (JSON.stringify(_this2.state[name]) !== JSON.stringify(prevState[name])) {
```
Comment (ktnyt) [inline modules/Endpoints.js]: If I understand correctly `_this2` should be `this`.

File: modules/Endpoints.js [hunk 2]
```diff
@@ -118,10 +115,14 @@ class Endpoints extends Component {
...
+        fillDefaults(configs[name]).onChange(data[name])
+      }
     })
 
-    onChange(data)
+    if (JSON.stringify(_this2.state) !== JSON.stringify(prevState)) {
```
Comment (ktnyt) [inline modules/Endpoints.js]: If I understand correctly `_this2` should be `this`.

**Review by ktnyt — COMMENTED** [review]
Overall the changes looks fine but the `_this2` looks like they are artifacts from babel transform. Can you give me a feedback on whether this is intended?

**Comment by s-horiguchi** [comment]
@ktnyt You are right.
Please check the latest one.

Another bug is also fixed.

**Comment by ktnyt** [comment]
LGTM

**Pull request merged**